src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/SexGenotypeContigPairKey.java

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.hellbender.tools.exome.germlinehmm;
+
+import org.broadinstitute.hdf5.Utils;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+
+/**
+ * This class provides a key made of the String pair (sex genotype, contig).
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class SexGenotypeContigPairKey implements Serializable {
```

Comment:
... but it _is_ only used as a key. Doesn't it make the code more readable to assert the use case of this class in its name? I don't have an opinion here, it's your call. Just put a comment.